debian/copyright

```diff
@@ -369,6 +369,14 @@ Files: third_party/arrow/abi.h
 Copyright: 2016-2024 The Apache Software Foundation
 License: Apache-2.0
 
+Files: third_party/nanoarrow/flatcc/*
+Copyright: 2015-2023 Mikkel F. Jørgensen, dvide.com
```

Comment:
We don't use non-ASCII characters in our text files. Please replace `ø` with `o`.

Reply:
Removed this line, because now nanoarrow is imported as a submodule.